### PR TITLE
LPS-157997 | Master - Bravo Team

### PR DESCRIPTION
**Pull request by MirnaGama**
### Motivation
The address is not showing above the map anymore when editing a geolocation fragment from a web content. 7.2 version has this feature working just fine. It was discovered that the files used in 7.2 are no longer the same used as in the latest version.

### Proposed Solution
New implementation to create a label for address and add the address information on the HTML along with the Clay Icon for Geolocation.

### Steps to Verify
1. Set up a vanilla bundle
2. Startup and create a web content structure with a geolocation
3. Create a web content with the new structure
4. Set a location and observe that the location as text is not appearing

### Related Issue
https://issues.liferay.com/browse/LPS-157997

### Commits Included
- LPS-157977 Show address as text on Geolocation Web Content Edit Page
- LPS-157977 Replace Glyphicon with ClayIcon
- LPS-157977 SF

### CODE REVIEW

**Comment by MirnaGama** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Geolocation/Geolocation.es.js [hunk 1]
```diff
@@ -55,6 +56,12 @@ const Geolocation = ({
...
 		<div {...otherProps} className="ddm-geolocation field-labels-inline">
+			<div>
+				<ClayIcon symbol="geolocation" />
+
+				<span id={`address_label_${instanceId}`}></span>
+			</div>
+
 			{!disabled || viewMode ? (
 				<dl>
 					<dt className="text-capitalize"></dt>
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Geolocation/Geolocation.es.js:67]: Hi @MirnaGama,

Can you move this `div` to be shown only if `!disabled || viewMode`? This way we will have the same behavior as 7.2.x version.

Something like:

```
{!disabled || viewMode ? (
	<>
		<div>
			<ClayIcon symbol="geolocation" />

			<span id={`address_label_${instanceId}`}></span>
		</div>

		<dl>
			<dt className="text-capitalize"></dt>

			<dd>
				<NoRender
					className="lfr-map"
					id={`map_${instanceId}`}
					style={{height: '280px'}}
				/>

				<input
					id={`input_value_${instanceId}`}
					name={name}
					type="hidden"
				/>
			</dd>
		</dl>
	</>
)
```

Thanks!

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by carolmariaabb** [comment]
@miltonmc could you please also review this PR? Thanks!

**Review by MirnaGama — COMMENTED** [review]
LGTM

**Comment by MirnaGama** [comment]
@carolmariaabb 
It's done !

**Review by miltonmc — APPROVED** [review]
LGTM!

**Comment by miltonmc** [comment]
ci:test:relevant

**Comment by MirnaGama** [comment]
Oops! My bad :sweat_smile:

**Comment by MirnaGama** [comment]
ci:test:sf

**Comment by MirnaGama** [comment]
ci:test:relevant

**Comment by MirnaGama** [comment]
ci:test:sf

**Comment by MirnaGama** [comment]
ci:test:relevant

**Comment by MirnaGama** [comment]
ci:test:sf

**Comment by miltonmc** [comment]
ci:forward

**Comment by miltonmc** [comment]
ci:reevaluate:1327401_4823

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by miltonmc** [comment]
ci:forward

**Pull request closed without merging**